### ISSUE TITLE
[Feature] Show unique findings on findings page

### ISSUE THREAD

**Issue by zcrt** [minvws/nl-kat-coordination#538]
**Is your feature request related to a problem? Please describe.**
The findings page can get cluttered quite quickly. 

**Describe the solution you'd like**
It would be helpful if there is a list of all unique findings available on the findings page (instead of first seeing 20 times the same finding).

**Comment by zcrt** [comment]
Yes I think you are correct. The idea is the same as a hyperlinked version of 2.3 ("Bevinding Types") in the bevindingenrapport. 
Currently I there is a finding that for some reason gets repeated 40 times (like expired certificates), you lose the overview of all finding types. If there is a list on top where you can click/filter on the finding type it would already be an improvement

**Comment by underdarknl** [comment]
Ok, we can work on that I think.
So adding the same summary above the list, or using the same accordeon style table would work I'd say.
